lua/net.lua

```diff
@@ -22,6 +22,8 @@ return function (gatekeeper_server)
 	-- passed to any call of getradom(2). This is relevant for production
 	-- environments to guarantee entropy while machines are booting up.
 	local guarantee_random_entropy = 0
+	local num_attempts_link_get = 5
+	local ipv6_default_hop_limits = 255
```

Comment:
Since we are moving hidden, undocumented parameters into runtime, we must document what they do and their units. If possible, give some suggestion on how to set them, or describe their impact.

This comment applies to all patches, not only here.